.github/hacktoberfest.md

```diff
@@ -0,0 +1,24 @@
+# Hacktoberfest
+
+During October 2022, Astro’s docs repo will accept your PRs for [Hacktoberfest](https://hacktoberfest.com/). Here’s how to take part!
+
+1. Make sure you’ve [registered with the Hacktoberfest organizers](https://hacktoberfest.com/).
+
+2. Open PRs to this repo between October 1 and October 31. (The earlier the better!)
+
+3. Let us know you’re taking part by requesting for your PR to be labelled as `hacktoberfest-accepted`.
+
+4. Have your PR merged by 31 October 2022.
```

Comment:
I get that this is a step, but the merging thing isn't really on them. So, can we do something that more lets them know what THEY can do to help US merge in time?

"Have your PR ready in time so that you can address all review comments and it can be approved and merged by 31 October 2022"  ??

Reply:
Maybe it’s better not to have this as a numbered step then. Just a final note:

```suggestion
Remember, we need to accept your PR by 31 October 2022 for it to count towards Hacktoberfest. Have your PR ready as early as possible, so that you can address comments and have it approved before the deadline.
```

(Leaving out a mention of a “merge” as it seems that labelling not merging is what’s actually key based on [the Hacktoberfest docs](https://hacktoberfest.com/participation/#pr-mr-details).)

Reply:
Ooh, good point! I did see that re: not needing to merge. Approve!